web/src/beta/features/PluginPlayground/Plugins/index.tsx

```diff
@@ -38,26 +39,88 @@ const Plugins: FC<Props> = ({
   sharedPlugin,
   handlePluginDownload
 }) => {
+  const t = useT();
   const [isAddingNewFile, setIsAddingNewFile] = useState(false);
 
   const handlePluginShare = (): void => {
     if (!selectedPlugin) return;
     encodeAndSharePlugin(selectedPlugin.id);
   };
 
+  const getCategoryTitle: Record<string, string> = useMemo(() => {
```

Comment:
```suggestion
  const categoryTitles: Record<string, string> = useMemo(() => {
```
getXxx usually means it's a function, since here it's a record maybe better to rename it to avoid any possible misunderstanding.

Reply:
Please update the naming of other translation records as well.

Reply:
Thanks. I've worked on this.